web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -63,9 +79,30 @@ export const createAppConfigSchema = (platform: Platform) => {
       .describe(
         '`auto` uses modern ConPTY system if available, which requires Windows 10 (19H1) or above. Set to `winpty` to use winpty even if ConPTY is available.'
       ),
+    'terminal.shell': z
+      .string()
+      .default(settings.defaultOsShellId)
+      .describe(
+        'A default terminal shell. It is best to configure it through UI (right click on a terminal tab > Default Shell).'
+      )
+      .refine(
+        configuredShell =>
+          availableShellIdsWithCustom.some(
+            shellId => shellId === configuredShell
+          ),
+        configuredShell => ({
+          message: `Cannot find the shell "${configuredShell}". Available options are: [${availableShellIdsWithCustom.join(', ')}]. Using platform default.`,
```

Comment:
This renders the available options like an array, wouldn't it be tidier to just separate them with comma?

```suggestion
          message: `Cannot find the shell "${configuredShell}". Available options are: ${availableShellIdsWithCustom.join(', ')}. Using platform default.`,
```

<img width="338" alt="unknown-shell" src="https://github.com/user-attachments/assets/54d149df-feb1-4c0d-87b7-677999e67805">